index.js

```diff
@@ -27,13 +28,21 @@ const {
 
 const app = express();
 const port = PORT || 3000;
+const sess = {
+  secret: GRAVITY_SECRET,
```

Comment:
This doesn't _have_ to be the client application secret token, it can just be any 'secret'. Not sure if there's a downside to reusing it.